bot.go

```diff
@@ -193,16 +193,20 @@ func eval(input string, formatMode bool, compactMode bool) string {
 			if compactMode {
 				res = compactModeStr
 			}
-			res += "\n```go\n" + formatted + "```is\n"
+			res += "\n```go\n" + formatted + "``` produces: "
+		}
+		evalres = strings.TrimSpace(evalres)
+		if evalres != "" {
+			res += "```go\n" + evalres + "\n```\n"
```

Comment:
You should create a function to format fenced Markdown block for go this way code will be way easier to read

````go
func formatMarkdownBlock(lang string, content string) string {
    return fmt.Errorf("```%s\n%s"\n```\n", lang, strings.TrimSpace(content) + "
}
```

So here, and everywhere you would use


```suggestion
			res += formatMarkdownBlock("go", evalres)
```

And later

formatMarkdownBlock("diff", diff)

Reply:
good idea, #49